languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -41914,6 +41914,32 @@ though, and has since been slightly modified)-->
         </rule>
     </category>
     <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+        <rulegroup id="DE_MA_FAUTE" name= "utiliser -ma faute- plutôt que -de ma faute-">
+            <rule>
+                <pattern>
+                    <token regexp="yes">pas|rien|jamais|cas|point|tout</token>
+                    <marker>
+                        <token>de</token>
+                        <token regexp="yes">[mts]a|[nv]otre|leur</token>
+                        <token>faute</token>
+                    </marker>
+                </pattern>
+                <message>Préférez <suggestion><match no="3"/> <match no="4"/></suggestion>.</message>
```

Comment:
The message should be a little more explanatory – why is this preferrable? Also, if you use the tokens as they are, you can use the shorter notation:

`<message>Dans des contextes plus formels, il est préférable d'écrire <suggestion>\3  \4</suggestion>.</message>`

Reply:
'de'